src/storage/mvcc/reader/scanner/forward.rs

```diff
@@ -598,10 +601,26 @@ impl<S: Snapshot> ScanPolicy<S> for DeltaEntryPolicy {
             } else {
                 Ok((vec![], vec![]))
             };
+            let old_value = if self.extra_op == ExtraOp::ReadOldValue
```

Comment:
Does it affect incremental backup?

Reply:
We should set `self.extra_op` to `ExtraOp::Noop` in incremental backup.